src/components/button-swap/button-swap.stories.mdx

```diff
@@ -0,0 +1,163 @@
+  argTypes={{
+    content_start_icon: {
+      type: { name: 'string' },
+      control: {
+        type: 'select',
+        options: ['bell', 'magnifying-glass'],
+      },
+    },
+    first_button_visually_hidden_text: { type: { name: 'string' } },
+    first_button_label: { type: { name: 'string' } },
+    second_button_visually_hidden_text: { type: { name: 'string' } },
+    second_button_label: { type: { name: 'string' } },
+  }}
+/>
+
+# Button Swap
+
+The Button Swap component was built with the article Push Notifications use case in mind. It has two buttons that are swapped on click representing two visual states accompanied with visually hidden text for a more inclusive experience.
+
+You have the ability to pass `firstBtnCallback` and `secondBtnCallback` callback functions to the `initButtonSwap` initialization function to add custom actions for when each of the buttons is clicked.
+
+<Canvas>
+  <Story
+    name="Default"
+    inline={false}
+    args={{
+      first_button_visually_hidden_text: 'Notifications have been turned off.',
+      first_button_label: 'Give me all the notifications!',
+      second_button_visually_hidden_text: 'Notifications have been turned on.',
+      second_button_label: 'Just kidding, stop notifications, please.',
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: makeTwigInclude(
+            '@cloudfour/components/button-swap/button-swap.twig',
+            {
+              first_button_visually_hidden_text:
+                'Notifications have been turned off.',
+              first_button_label: 'Give me all the notifications!',
+              second_button_visually_hidden_text:
+                'Notifications have been turned on.',
+              second_button_label: 'Just kidding, stop notifications, please.',
+            }
+          ),
+        },
+      },
+    }}
+  >
+    {(args) => buttonSwapStory(args)}
+  </Story>
+</Canvas>
+
+## Template Properties
+
+- `class` (string): Appends a CSS class(es) to the root element.
+- `content_start_icon` (string, default `'bell'`): The button [icon](/docs/design-icons--page) (the same icon is applied to both buttons).
+- `first_button_visually_hidden_text` (string, default `'Unsubscribed from notifications'`): The first button visually hidden text.
+- `first_button_label` (string, default `'Get notifications'`): The first button label.
+- `second_button_visually_hidden_text` (string, default `'Subscribed to notifications'`): The second button visually hidden text.
+- `second_button_label` (string, default `'Turn off notifications'`): The second button label.
+
+## JavaScript
+
+You'll need to use JavaScript to select the Button Swap component via the `js-c-button-swap` CSS class and initialize it using the `initButtonSwap` function.
+
+### Syntax
+
+```js
+initButtonSwap(buttonSwapEl, options);
+```
+
+### Parameters
+
+#### `buttonSwapEl`
+
+The `.js-c-button-swap` Button Swap wrapper element.
```

Comment:
The `c-` component is not necessary for `js-` selectors. See elastic text areas for a comparison: https://v-next--cloudfour-patterns.netlify.app/?path=/docs/components-input--elastic-textarea#javascript-instructions

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1516/commits/47177615cbb93df03875f2b536808393242d5cc7